drivers/hmis_csv_twenty_twenty/app/models/hmis_csv_twenty_twenty/importer/organization.rb

```diff
@@ -35,5 +35,25 @@ def self.warehouse_class
     #     date_range: date_range,
     #   )
     # end
+
+    def self.hmis_validations
+      {
+        OrganizationID: [
```

Comment:
Maybe always require the UID? I think we could do that in `Base`...

Reply:
Not a terrible idea, let's do that as a second pass.